cfgov/jinja2/v1/youth_employment_success/review/print.html

```diff
@@ -0,0 +1,5 @@
+<div class="block__sub">
+  <p class="h3">Print your plan</p>
+  <p>You can print your plan as a PDF to keep track of your options and next steps</p>
+  <button class="a-btn" type="button">Print your plan</button>
```

Comment:
Do you need `type="button"` attribute here?

Reply:
I guess I thought that annotation would be required for it to be valid for screen readers etc.